docs/05.1-entities.mdx

```diff
@@ -3,16 +3,18 @@ title: Entities
 path: /docs/entities
 ---
 
-An [[Entity]] in is an empty container for [[Components]], alone an Entity has little behavior.
+An [[Entity]] in is an empty container for [[Components]]. On its own, an Entity has very little behavior.
 
-They are useful for defining bare bones game objects with custom behavior.
+Entities are useful for defining basic game objects with custom behavior.
 
-If you want a batteries included prebuilt Entity use an [[Actor]], read more [here](/docs/actor).
+If you want a preconfigured Entity, use an [[Actor]] (read more [here](/docs/actor)).
 
 ## Entity
 
+//TODO clarify
```

Comment:
How about:

"Entities have no behavior by default, they are only containers for components. They also participate in `onInitialize`, `onPreUpdate`, and `onPostUpdate` lifecycle hooks."